### ISSUE TITLE
<b-card> Add custom css for header and footer

### ISSUE THREAD

**Issue by Mike-Yan** [bootstrap-vue/bootstrap-vue#1900]
Hi I saw in the doc you can change the background color of the header by using `header-bg-variant` `primary` `success` etc. as a variable 

This is a default template from the doc.
```
   <b-card header="Card Header"
                        header-text-variant="white"
                        header-tag="header"
                        header-bg-variant="primary"
                        title="Title"
                        style="max-width: 20rem "
                >
```
How do I change the background-color? I checked this #767 and tried `.card-card_header: {...}` ` header-variant="card_header"`doesn't work for me.

Any solution to this or did I miss something in the doc?

**Comment by Mike-Yan** [comment]
![screen shot 2018-06-22 at 09 37 37](https://user-images.githubusercontent.com/16896764/41763963-faf85f34-75ff-11e8-8214-f15a30a09268.png)

Thank you for responding me !\
\
Yes I already tried something like that before, but I still have the same problem. As you can see in the screenshot the whole border is not filled. <br/><br/> I can overwrite `background-color ` in the css class<br/> 
``` 
 .card-header {
    padding: 0.75rem 1.25rem;
    margin-bottom: 0;
    background-color: #fff;
    border-bottom: 1px solid rgba(0, 0, 0, 0.125);
  }
``` 
<br/>
What I am trying to do is making a dynamic css.  Like this

 ``` 
data() {
  return {
    colorStyles: {
      loggedin: {
        'background-color': ''red"
      }

**Comment by mosinve** [comment]
You should take into account this:

> If you’ll examine b-card rendered html, then you’ll see that `header` slot is a header content actually. It’s not completely replace header, but it wrapped around by `div.card-header` or whatever tag you specify by `header-tag` prop

**Comment by Mike-Yan** [comment]
Yes I saw it in inspect element that it's wrapped by `card-header`. I am not that experienced with Front-End and new to VueJS. As you said it's wrapped around by `div.card-header`. I am not sure how to write the css in vue code instead of overwriting in the css scope for the header part.  I will keep trying and thanks for your replies !

**Comment by Mike-Yan** [comment]
I am not that experienced with front-end developing and especially new to VueJS. But this seems the solution for me.
```
.my-header {
  padding: 0;
}
```
I want to thank you for the replies and help ! 😄
